test/lib.sh

```diff
@@ -16,6 +16,10 @@
 
 # Include after test-infra/scripts/library.sh
 
+# Set and export SYSTEM_NAMESPACE where Knative Eventing is installed for pkg/system.
+readonly SYSTEM_NAMESPACE="knative-eventing"
```

Comment:
I guess the eventing controllers need this right? Can we update the yaml to inject the namespace? This is the recommended approach in pkg/system

Reply:
It's my understanding that the eventing controllers already have injected SYSTEM_NAMESPACE in the way you suggest. What has changed is that now also the eventing test library requires access to SYSTEM_NAMESPACE. Does this still mean we should be injecting via YAML?

In eventing e2e tests, SYSTEM_NAMESPACE is set in the shell script: https://github.com/knative/eventing/blob/master/test/e2e-common.sh#L68
And downstream as well: https://github.com/knative-sandbox/eventing-kafka/blob/master/test/e2e-tests.sh#L69

Reply:
Ah I see, looks like the script actually depends on this variable. Ideally we only have one source of truth (in the yaml) but that's out of scope.